doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,71 +210,139 @@ infrastructure itself from the workloads compute resources.
+    - The integration of SDN solutions required by `req.inf.ntw.05` is enabled
+via CNI integration.
+    - IP Address Management (**IPAM**) of the various networks can be provided
+by one or more IPAM plugins, which can be part of a CNI plugin, or some other
+component (i.e. external SDN solution) - it is key that there are no overlapping
+IP addresses within a cluster, and if multiple IPAM solutions are used that
+they are co-ordinated in some way (as required by `req.inf.ntw.10`).
+- **Service Mesh**: The well known service meshes are "application service meshes"
+that address and interact with the application layer 7 protocols (eg.: HTTP)
+only. Therefore, their support is not required in this architecture, as these
+service meshes are outside the scope of the infrastructure layer of this
+architecture.
+
+<p align="center"><img src="../figures/ch03_networking.png" alt="Kubernetes Networking Architecture" Title="Kubernetes Networking Architecture" width="100%"/></p>
+<p align="center"><b>Figure 3-2:</b> Kubernetes Networking Architecture</p>
+
+<!--The above diagram is maintained here:
+https://wiki.lfnetworking.org/display/LN/CNTT+RA2+-+Kubernetes+-+Diagrams+-+Networking-->
+
+There are several types of low latency and high throughput networks required by
 telco workloads: signalling traffic workloads and user plane traffic workloads.
 Networks used for signalling traffic are more demanding than what a standard
 overlay network can handle, but still do not need the use of user space
 networking. Due to the nature of the signalling protocols used, these type of
-networks require NAT-less communication documented in infra.net.cfg.003 and will
+networks require NAT-less communication documented in `infra.net.cfg.003` and will
 need to be served by a CNI plugin with IPVLAN or MACVLAN support. On the other
 hand, the low latency, high throughput networks used for handling the user plane
 traffic require the capability to use a user space networking technology.
 
+| Requirement | Networking Solution with Multus | Networking Solution with DANM | Networking Solution with Tungsten Fabric | Networking Solution with NSM |
+|---|---|---|---|---|
+| The overlay network encapsulation protocol needs to enable ECMP in the underlay (`infra.net.cfg.002`) | Supported via the additional CNI plugin | Supported via the additional CNI plugin | Supported | TBC |
+| NAT (`infra.net.cfg.003`) | Supported via the additional CNI plugin | Supported | Supported | TBC |
+| Network Policies (Security Groups) (`infra.net.cfg.004`) | Not supported | Not supported <sub>(1)<sub> | Supported | Supported via a CNI Network Plugin that supports Network Policies |
+| SFC support (`infra.net.cfg.005`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Traffic patterns symmetry (`infra.net.cfg.006`) | Not relevant | Not relevant | Not relevant | Not relevant |
+| Network resiliency (`req.inf.ntw.01`) | Supported | Supported | Supported | Supported |
```

Comment:
Is network resiliency supported by the solution or is supported by kubernetes

Reply:
It's not a great requirement tbh "The Architecture must support network resiliency." - could mean anything. Resilient DC fabric? Multiple NICs to each worker node? What sort of resiliency, and to mitigate which risks?  I think unless we have better requirements, I think the answer needs to remain fairly high level.

Reply:
@tomkivlin Should this requirement be dropped given that it raises more questions and the "supported" doesn't provide any insights.

Reply:
@pgoyal01 I think dropping requirements at this late stage isn't ideal.  I'd rather leave it as-is and deal with the removal or improvement of the requirement and any subsequent wording in ch3/ch4 post-Elbrus.

Reply:
@tomkivlin OK with a backlog issue

Reply:
#2187 created

Reply:
@tomkivlin Thanks for creating the issue. Can this entry be deleted from this table?

Reply:
Yes good idea, have done in https://github.com/cntt-n/CNTT/pull/2121/commits/fca7128df5da841af8a8366fe9be94b09c79a203